crates/bitwarden-exporters/src/cxf/editable_field.rs

```diff
@@ -0,0 +1,225 @@
+use bitwarden_vault::FieldType;
+use credential_exchange_format::{
+    EditableField, EditableFieldBoolean, EditableFieldConcealedString, EditableFieldString,
+    EditableFieldWifiNetworkSecurityType,
+};
+
+use crate::Field;
+
+/// Helper function to create a Field from any EditableField type
+pub(super) fn create_field<T>(name: String, field: &T) -> Field
+where
+    T: EditableFieldToField,
+{
+    Field {
+        name: Some(name),
+        value: Some(field.field_value()),
+        r#type: field.field_type() as u8,
+        linked_id: None,
+    }
+}
+
+/// Trait to convert EditableField types to Field values and types
+pub(super) trait EditableFieldToField {
+    fn field_value(&self) -> String;
+    fn field_type(&self) -> FieldType;
```

Comment:
Suggestion, `field_type` never uses `&self`, so I'd probably remove it from the trait. In fact it might be a better idea to change the definition to:
```rust
pub(super) trait EditableFieldToField {
    const FIELD_TYPE: FieldType;
    fn field_value(&self) -> String;
}
```

This way it's more clear that a field type will never change?

Reply:
It is, I initially thought we might at some point need to dynamically determine it but I can't think of a reason to do so beyond future proofing.